css/styles.css

```diff
@@ -42,38 +39,22 @@ h1 {
 
 .menu {
   width: 100%;
-  display: flex;
-  justify-content: space-around;
-  /* flex-direction: row; zu welchem elemnt?*/
```

Comment:
when you would like to use the flex-direction, it'll effect all items in the flexbox. so when you got "display: flex" on your menu you will choose every single item to be in the default setting: flex-direction: row; the only way to get one single element is to go into the element you want to place and choose align-self: "where you want".